Lesson1 (Sequence containers)/Proskuchenko/lesson1task2/magicNumber.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+#include <list>
+#include <cmath>
+
+size_t magicNumber(const size_t number, const size_t degree){
+  std::list<uint8_t> list_result;
```

Comment:
what the purpose of using the precise types? are they really needed?
static casts make your code unreadable